topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -58,23 +58,36 @@ We start with the question: In human chromosome 22, which exon has the highest n
+>
 >
 >   ![Eukaryotic gene model](../../images/DNA_exons_introns.png "An original piece of DNA containing introns and exons has the introns cut out before the exons are joined together to form the mRNA. (Image from <a href="https://commons.wikimedia.org/wiki/File:DNA_exons_introns.gif">WikiMedia</a>, under the Public Domain)")
 >
-> - **SNP** stands for [single-nucleotide polymorphism](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism) and is pronounced like the English word "snip". SNPs are single nucleotide differences between a sequenced individual compared to some reference sequence; where one individual might have an A, another could have a C in that position. Databases of SNPs have been created for many organisms and they include any single nucleotide deviation from the reference sequence which appears in some percentage of the population (e.g. >1%). These are often especially interesting to geneticists as the causes of certain inheritable diseases.
+> **SNP** is an abbreviation for [single-nucleotide polymorphism](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism). It is pronounced as "snip". SNPs are single nucleotide differences between a sequenced individual compared to some reference sequence; where one individual might have an A, another could have a C in that position. Databases of SNPs have been created for many organisms and they include any single nucleotide deviation from the reference sequence which appears in some percentage of the population (e.g. >1%). These are often especially interesting to geneticists as the causes of certain inheritable diseases.
 {: .comment}
 
 You may be familiar with the [UCSC Genome Browser](https://genome.ucsc.edu/) or another resource like it, and know that you can find the data there.
-But even with your data in hand, you stll have the question: "how do I actually compute this?" There is really a straightforward answer: **Galaxy**. So let's try it...
+But even with your data in hand, you still have the question: "how do I actually compute this?" There is really a straightforward answer: **Galaxy**. So let's try it...
+
+The following video shows all steps of this tutorial:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/D5HgJWdfOWw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Hey @nekrut this video seems to only have autogenerated captions. Would you be willing to correct those before we merge this?

Just FYI, we're working on a framework level solution for this, so this video might get moved later and integrated in a different way.

Reply:
I did not have time to edit captions yet. I need this merged by COVID training. I will try to get captions edited

Reply:
Looking forward to the infrastructure :). We will try to get captions done, but we need to move here I think, and get the training ready and others onboard. I'm worried the entry barrier is growing and growing and it's not easy to contribute easily anymore if we block PRs here.

Reply:
@bgruening this requirement is only there for videos embedded in the page. If it's part of the GTN directly, then it needs to meet the standards for inclusion like accessibility (e.g. how we check all images for alt text.)

We're not increasing the barrier for PRs at all. That hasn't changed in a long time, it's only because of this additional thing, the embedded video, that it's a concern.

Reply:
And the contribution barrier for video production of production captions was always there :( people just ignored it to the exclusion of a lot of people who could be part of our community.